### PR TITLE
First database version

### PR DESCRIPTION
**Pull request by v-stamenova**
> ❗️ This branch is based on the #25 registration-flows branch that is not currently merged 
> This is because the migrations introduced there were needed

In this branch the migrations create a database that has a model that looks like this
image
Will create a drawio file later on

The models are created as well in order to establish the relationships between them

For testing:
* Connection between each table especially the many-to-many relationship (`participants`) and the relationship between the models `User`, `Speaker`, `Presentation`

### CODE REVIEW

**Comment by TimKardol** [comment]
Should this be merged after or before the jetstream teams rework? Seems like there are double commits.

**Comment by v-stamenova** [comment]
> Should this be merged after or before the jetstream teams rework? Seems like there are double commits.

If the Jetstream/Spatie permissions rework is merged, then it can be merged without any issues

File: app/Models/Booth.php [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+{
+    use HasFactory;
+
+    /**
+     * Establishes the connection
+     * A booth belongs to a team
+     *
+     * @return BelongsTo
+     */
+    public function team(): \Illuminate\Database\Eloquent\Relations\BelongsTo
```
Comment (dwaard) [inline app/Models/Booth.php:19]: It is more readable if the `\Illuminate\Database\Eloquent\Relations\BelongsTo` qualifier is replaced by an `import` (PHPStorm shoud suggest this as well)

File: app/Models/Booth.php [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+
+class Booth extends Model
+{
+    use HasFactory;
+
+    /**
+     * Establishes the connection
+     * A booth belongs to a team
```
Comment (dwaard) [inline app/Models/Booth.php:15]: Stating the obvious: "Establishes the connection". Instead, describe it as an attribute. Suggestion: "The Team that owns this Booth"

File: app/Models/Presentation.php [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+use Illuminate\Database\Eloquent\Relations\BelongsToMany;
+use Illuminate\Database\Eloquent\Relations\HasMany;
+
+class Presentation extends Model
+{
+    use HasFactory;
+
+    /**
+     * Returns the room that the presentation is in
```
Comment (dwaard) [inline app/Models/Presentation.php:16]: Empty comment line here

File: app/Models/Presentation.php [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+     * @return BelongsTo
+     */
+    public function room(): BelongsTo
+    {
+        return $this->belongsTo(Room::class);
+    }
+
+    /**
+     * Returns the timeslot that the presentation is in
```
Comment (dwaard) [inline app/Models/Presentation.php:25]: ditto

File: app/Models/Presentation.php (hunk 3 above)
Comment (dwaard) [reply app/Models/Presentation.php:16]: Also, decide whether or not remove the "Returns the" phrase for these kinds of comments. Decide and then apply this consistently throughout the entire project (see `Booth.php`)

File: app/Models/Presentation.php [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+     * @return BelongsTo
+     */
+    public function timeslot(): BelongsTo
+    {
+        return $this->belongsTo(Timeslot::class);
+    }
+
+    /**
+     * Returns all participants that signed up for the presentation
```
Comment (dwaard) [inline app/Models/Presentation.php:34]: ditto

File: app/Models/Presentation.php [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+     * @return BelongsToMany
+     */
+    public function participants(): BelongsToMany
+    {
+        return $this->belongsToMany(User::class, 'participants', 'presentation_id', 'user_id');
+    }
+
+    /**
+     * Returns all speakers for the presentation
```
Comment (dwaard) [inline app/Models/Presentation.php:43]: ditto

File: app/Models/Room.php [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\HasMany;
+
+class Room extends Model
+{
+    use HasFactory;
+
+    /**
+     * Returns all the presentations that are in the room
```
Comment (dwaard) [inline app/Models/Room.php:14]: ditto (you know the drill, so I'll won't add more of these comments)

File: resources/views/livewire/auth-navigation-menu.blade.php [hunk 8]
```diff
@@ -43,7 +43,7 @@ class="bg-white dark:bg-gray-900 border-b border-gray-100 dark:border-gray-700 r
```
Comment (dwaard) [inline resources/views/livewire/auth-navigation-menu.blade.php:43]: Seems like I'm missing the "Contact" navigation link when I compare this with the responsive menu. Suggestion: turn this into a Blade component and specify the menu links in some array. Then you can loop over the array for both menus

File: resources/views/livewire/guest-navigation-menu.blade.php [hunk 9]
```diff
@@ -71,16 +72,16 @@ class="inline-flex items-center justify-center p-2 rounded-md text-gray-400 dark
...
                 {{ __('Speakers') }}
             </x-responsive-nav-link>
             <x-responsive-nav-link href="" {{--:active="request()->routeIs('dashboard')"--}}>
                 {{ __('Companies') }}
             </x-responsive-nav-link>
-            <x-responsive-nav-link href="" {{--:active="request()->routeIs('dashboard')"--}}>
+            <x-responsive-nav-link href="" href="{{ route('faq') }}" :active="request()->routeIs('faq')">
                 {{ __('FAQ') }}
             </x-responsive-nav-link>
             <x-responsive-nav-link href="" {{--:active="request()->routeIs('dashboard')"--}}>
```
Comment (dwaard) [inline resources/views/livewire/guest-navigation-menu.blade.php:87]: Why this "empty" link?

**Review by dwaard — APPROVED** [review]
Mainly comments about comments 😄

**Pull request merged**